packages/strapi-provider-upload-aws-s3/lib/index.js

```diff
@@ -9,18 +9,30 @@
 const _ = require('lodash');
 const AWS = require('aws-sdk');
 
+const getFilePath = (filePath, s3Prefix) => {
+  if (s3Prefix) {
+    return `${s3Prefix}${filePath}/`;
+  }
+  return filePath;
```

Comment:
return value in this case should be 

`return filePath ?`${filePath}/` : '';`

To respect original logic on line 23